ember-cli/2018-07/july-19.md

```diff
@@ -0,0 +1,7 @@
+# ember-cli Meeting Notes - 2018-07-19
+
+## Attendees
+
+## Agenda
```

Comment:
in no particular order:

+ https://github.com/ember-cli/ember-cli/pull/7895
+ https://github.com/ember-cli/ember-cli/pull/7897
+ https://github.com/ember-cli/ember-cli/pull/7910
+ https://github.com/ember-cli/ember-cli/pull/7458